### ISSUE TITLE
Access to Highcharts.getOptions().colors?

### ISSUE THREAD

**Issue by michaeljones** [kirjs/react-highcharts#6]
Hi,

I'm working through a Highcharts example and using React with your excellent project here.

Unfortunately I'm really struggling to access `Highcharts.getOptions().colors`. Can I do it from a standard import like:
```javascript
var Highcharts = require('react-highcharts');
```
Or do I have to import the highcharts library separately? I'm afraid my understanding of node's modules is not good enough to figure it out, though hopefully it is simple.

Any help would be much appreciated,
Michael

**Comment by Kureev** [comment]
`Highcharts` in this bundle is a `React.Component`, so it's not possible to access colors the way you specified above. But it could be an improvement for the future: expose `highcharts` static methods

**Comment by michaeljones** [comment]
Thanks for the response. I think it would be useful. Again my understanding of javascript modules is not that best but I'd be concerned that trying to include Highcharts in another way might result in having it twice which would impact the file size. (I'm using browserify.)

This might tie in to issue #4, I guess. Not sure.

**Comment by Kureev** [comment]
I'll try to take a look into it, otherwise @kirjs will fix it in a few weeks. Hope I'll be faster.

**Comment by kirjs** [comment]
Hi michaeljones,

You should be able to access Highcharts object as `Highcharts.Highcharts`.
e.g. `Highcharts.Highcharts.getOptions().colors`.

Please let me know if this works for you.

**Comment by michaeljones** [comment]
Hi @kirjs,

Thanks for the response and thank you @Kureev. I did try your approach @kirjs and it didn't seem to work but I've just realised I was still on `0.0.7`. If I update to the latest on npm then it works.

Sorry for the trouble, I should know better than to fail to check for newer versions.

It might be worth documenting the availability of `Highcharts.Highcharts` if it isn't covered already. I can't see reference to it.

Cheers,
Michael
